### PR TITLE
fix(run/websocket): Debug Websocket test errors

### PR DESCRIPTION
**Pull request by averikitsch**
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/commit/05b5423242f273518e8600e26e2bd35036082959#r61378446

### CODE REVIEW

File: run/websockets/package.json [hunk 1]
```diff
@@ -12,12 +12,12 @@
   "dependencies": {
     "@socket.io/redis-adapter": "^7.0.0",
     "express": "^4.17.1",
-    "redis": "^4.0.0",
+    "redis": "^3.1.2",
```
Comment (ace-n) [inline run/websockets/package.json:15]: Is there a way to avoid downgrading this?

Comment (averikitsch) [reply run/websockets/package.json:15]: There is a breaking change in the library. To avoid rewriting the entire sample this is needed.

Comment (ace-n) [reply run/websockets/package.json:15]: Got it. Is this something we plan to fix separately?

Comment (averikitsch) [reply run/websockets/package.json:15]: https://github.com/GoogleCloudPlatform/nodejs-docs-samples/issues/2657

**Review by ace-n — APPROVED** [review]

**Review by sofisl — APPROVED** [review]

**Pull request merged**